tutorials/getting_started/creating_a_model.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: tutorial
+title: Creating Your Own Models
+id: creating-a-model
+---
+
+Using the included models is fine, but at some point you'll probably want to implement your own models,
+which is what this tutorial is for.
```

Comment:
I think it'd be nice to say somewhere near the beginning that there typically just two classes you have to implement in order to build your own model on some new task: (1) a dataset reader, and (2) a model.  If you're just building a new model for an existing task, you don't even need to do (1).

Reply:
done